src/routes/routes.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react'
+import { View, ActivityIndicator } from 'react-native'
+import {useAuth} from '../contexts/auth'
+import AppRoutes from './app.routes'
+import AuthRoutes from './auth.routes'
+
+
+const Routes: React.FC = () => {
```

Comment:
nitpick: we could use typescript inference (same about others components)

```suggestion
const Routes = () => {
```